src/renderer/components/TranscriptionBlock.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/styled';
+import { Box, colors } from '@mui/material';
+import { Transcription } from 'sharedTypes';
+
+const TranscriptionBox = styled(Box)`
+  background: ${colors.common.white};
+  height: 100%;
+  padding: 20px;
+`;
+
+const Word = styled('span')`
+  &:hover {
+    background: orange;
+  }
+`;
+
+interface Props {
+  transcription: Transcription;
+  onWordClick: (wordIndex: number) => void;
+}
+
+const TranscriptionBlock = ({ onWordClick, transcription }: Props) => {
+  return (
+    <TranscriptionBox>
+      <p>
+        {/* eslint-disable-next-line prettier/prettier */}
+        {transcription.words.map((word, index) => (
+          !word.deleted ? (
```

Comment:
Flip this ternary so the else isn't a double negative - or better yet, just filter the deleted words before mapping

Reply:
@rileykeane are you changing it to filter before mapping?

Reply:
yep. although won't it still be a double negative using filter? 

`transcription.words.filter((word) => !word.delete).map(...)`

Am I missing something here?

Reply:
Nope, because if we use filter we don't need the ternary :)

Reply:
Ternary has been flipped. Using filter sounds nice on paper but ended up being more difficult than it's worth.